modules/ROOT/pages/execution-plans/operators.adoc

```diff
@@ -2575,7 +2575,7 @@ Total database accesses: 215, total allocated memory: 3440
 == VarLength Expand All
 // VarLengthExpand(All)
 
-Given a start node, the `VarLengthExpand(All)` operator will traverse variable-length relationships.
+Given a start node, the `VarLengthExpand(All)` operator will traverse variable-length/quantified relationships.
```

Comment:
The same suggestion applies for the below examples of "variable-length/quantified".
```suggestion
Given a start node, the `VarLengthExpand(All)` operator will traverse variable-length and quantified relationships.
```